docs/pages/guides/config-plugins.mdx

```diff
@@ -4,25 +4,26 @@ title: Config Plugins
 
 import { Terminal } from '~/ui/components/Snippet';
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
-
-> This guide applies to SDK 41+ projects. The Expo Go app doesn't support custom native modules.
+import { Collapsible } from '~/ui/components/Collapsible';
 
 When adding a native module to your project, most of the setup can be done automatically by installing the module in your project, but some modules require a more complex setup. For instance, say you installed `expo-camera` in your bare project, you now need to configure the native app to enable camera permissions — this is where config plugins come in. Config plugins are a system for extending the Expo config and customizing the prebuild phase of managed builds.
 
 Internally Expo CLI uses config plugins to generate and configure all the native code for a managed project. Plugins do things like generate app icons, set the app name, and configure the **Info.plist**, **AndroidManifest.xml**, etc.
 
-You can think of plugins like a bundler for native projects, and running `expo prebuild` as a way to bundle the projects by evaluating all the project plugins. Doing so will generate `ios` and `android` directories. These directories can be modified manually after being generated, but then they can no longer be safely regenerated without potentially overwriting manual modifications.
+You can think of plugins like a bundler for native projects, and running `npx expo prebuild` as a way to bundle the projects by evaluating all the project plugins. Doing so will generate `ios` and `android` directories. These directories can be modified manually after being generated, but then they can no longer be safely regenerated without potentially overwriting manual modifications.
 
 #### Quick facts
 
 - Plugins are functions that can change values on your Expo config.
-- Plugins are mostly meant to be used with [`expo prebuild`][cli-prebuild] or `eas build` commands.
+- Plugins are mostly meant to be used with [`npx expo prebuild`][cli-prebuild] or `eas build` commands.
 - We recommend you use plugins with **app.config.json** or **app.config.js** instead of **app.json** (no top-level `expo` object is required).
 - `mods` are async functions that modify native project files, such as source code or configuration (plist, xml) files.
 - Changes performed with `mods` will require rebuilding the affected native projects.
 - `mods` are removed from the public app manifest.
 - Everything in the Expo config must be able to be converted to JSON (with the exception of the `mods` field). So no async functions outside of `mods` in your config plugins!
 
+> **Note**: The Expo Go app doesn't support custom native modules, and config plugins so not apply there.
```

Comment:
```suggestion
> **Note**: The Expo Go app doesn't support custom native modules, and config plugins so not apply there.
```

This reads a little confusing when said "so not apply there". Is it saying that config plugins are not applicable there because the Expo Go app doesn't support custom native modules?